package.json

```diff
@@ -56,23 +56,28 @@
     "@types/react": "^17.0.14",
     "@types/react-dom": "^17.0.9",
     "@types/styled-components": "^5.1.6",
+    "@typescript-eslint/eslint-plugin": "^4.31.2",
     "@walletconnect/client": "^1.6.1",
     "autoprefixer": "^10.3.1",
     "babel-loader": "^8.2.2",
     "cssnano": "^5.0.7",
-    "eslint-plugin-prettier": "^3.4.0",
+    "eslint": "^7.32.0",
+    "eslint-config-rainbow": "^2.0.0",
+    "eslint-plugin-jest": "^24.4.2",
```

Comment:
Not sure why but plugin-jest was not resolving correctly from rainbow or tsdx, so I added it top level.  First time seeing this issue, if there is something I can do in eslint-config-rainbow I'll add a fix there.